kindcat_custom.cue

```diff
@@ -63,7 +63,11 @@ _crdSchema: {
 		// Have to do this gnarly regex instead
 		[!~"^(uid|creationTimestamp|deletionTimestamp|finalizers|resourceVersion|labels|updateTimestamp|createdBy|updatedBy|extraFields)$"]: string
 	}
-	spec: struct.MinFields(0)
+	spec: _
```

Comment:
is _ an empty placeholder in cue (similar to JS empty, unused param in a function?)

Reply:
it's "top", most closely analogous to `any` in typescript